src/app/pokemon-list/pokemon-list.component.ts

```diff
@@ -46,6 +55,7 @@ export class PokemonListComponent implements OnInit {
       .subscribe(pokemons => {
         this.pokemons = pokemons;
         this.loading = false;
+        this.cdRef.detectChanges();
```

Comment:
Never use detectChanges if u want to update data in async context

Reply:
0acf1f06cbd361cd209c3f1c0ca1c1f3cb208437